sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -17,6 +17,7 @@ public abstract partial class AsyncPageable<T> : System.Collections.Generic.IAsy
     }
     public static partial class AzureCoreExtensions
     {
+        public static Azure.Messaging.CloudEvent? ToCloudEvent(this System.BinaryData binaryData) { throw null; }
```

Comment:
It is debatable whether this belongs here or left in Event Grid.

Reply:
> Then it wouldn't show up on every BinaryData instance just because you had a using Azure; statement in your project?

It would. VS shows extensions from all namespaces.

What bout `CloudEvent.FromBinaryData()`?

Reply:
We already have a Parse method that takes a string. We could potentially have a parse from BinaryData as well.

Reply:
Ended up changing to CloudEvent.Parse.